tests/lib/rules/indent.js

```diff
@@ -682,6 +682,12 @@ ruleTester.run("indent", rule, {
                 "  }\n" +
                 "};",
             options: [2]
+        }, {
+            code:
+                "var path     = require('path')\n" +
+                " , crypto    = require('crypto')\n" +
+                " ;\n",
```

Comment:
No, it has `2` spaces before it (excluding comma) and the config also says `2` spaces so I think this is normal. Also, even if that part was unexpected, this patch only fixes the behavior for the semicolon, not anything else.